sdk/batch/azure-compute-batch/src/main/java/com/azure/compute/batch/BatchClientBuilder.java

```diff
@@ -236,27 +217,30 @@ public BatchServiceClientBuilder serviceVersion(BatchServiceVersion serviceVersi
      * Sets The retry policy that will attempt to retry failed requests, if applicable.
      *
      * @param retryPolicy the retryPolicy value.
-     * @return the BatchServiceClientBuilder.
+     * @return the BatchClientBuilder.
      */
     @Generated
-    public BatchServiceClientBuilder retryPolicy(RetryPolicy retryPolicy) {
+    public BatchClientBuilder retryPolicy(RetryPolicy retryPolicy) {
         this.retryPolicy = retryPolicy;
         return this;
     }
 
     /**
-     * Builds an instance of BatchServiceClientImpl with the provided parameters.
+     * Builds an instance of BatchClientImpl with the provided parameters.
      *
-     * @return an instance of BatchServiceClientImpl.
+     * @return an instance of BatchClientImpl.
      */
     @Generated
-    private BatchServiceClientImpl buildInnerClient() {
+    private BatchClientImpl buildInnerClient() {
         HttpPipeline localPipeline = (pipeline != null) ? pipeline : createHttpPipeline();
         BatchServiceVersion localServiceVersion =
                 (serviceVersion != null) ? serviceVersion : BatchServiceVersion.getLatest();
-        BatchServiceClientImpl client =
-                new BatchServiceClientImpl(
-                        localPipeline, JacksonAdapter.createDefaultSerializerAdapter(), endpoint, localServiceVersion);
+        BatchClientImpl client =
+                new BatchClientImpl(
+                        localPipeline,
+                        JacksonAdapter.createDefaultSerializerAdapter(),
+                        this.endpoint,
+                        localServiceVersion);
         return client;
     }
 
```

Comment:
Not a blocker for this PR but since the **createHttpPipeline** is an autogenerated method, we should consider extending it with our own manual method that way we don't lose any update changes to this method due to the removal of @Generated tag.

Reply:
Looking at the diff, I don't see that the **@Generated** tag present in the diff at all for **createHttpPipeline**. I'm guessing it was never there in the first place?